src/frontends/onnx/frontend/src/op/com.microsoft/range.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2018-2024 Intel Corporation
```

Comment:
Please, use 2025 for now :) Happy new year!

Reply:
done.
Happy new year :)